pymc3/gp/gp.py

```diff
@@ -136,8 +136,8 @@ def sample_gp(trace, gp, X_values, samples=None, obs_noise=True, model=None, ran
     # Posterior covariance
     S_post = S_zz - tt.dot(tt.dot(S_xz.T, S_inv), S_xz)
 
-    gp_post = MvNormal.dist(m_post, S_post, shape=Z.shape[0])
-    
+    gp_post = MvNormal.dist(m_post, S_post + jitter * tt.eye(S_post.shape[0]), shape=Z.shape[0])
```

Comment:
Not sure "jitter" is the right name, though. It implies randomness, like what you add to a plot to make overlapping points visible. This is just a small constant, so maybe `cov_const` or similar?